### PR TITLE
apps: move deployer to staging

### PR DESCRIPTION
**Pull request by mfojtik**
Clean move as we ripped the edges already (apps util).

/cc @soltysh 
/cc @deads2k

### CODE REVIEW

File: staging/src/github.com/openshift/oc/pkg/cli/deployer/deployer.go [hunk 1]
```diff
@@ -26,11 +26,11 @@ import (
...
 	"github.com/openshift/library-go/pkg/apps/appsutil"
-	"github.com/openshift/origin/pkg/apps/strategy"
-	"github.com/openshift/origin/pkg/apps/strategy/recreate"
-	"github.com/openshift/origin/pkg/apps/strategy/rolling"
-	cmdversion "github.com/openshift/origin/pkg/cmd/version"
-	"github.com/openshift/origin/pkg/version"
+	"github.com/openshift/oc/pkg/cli/deployer/strategy"
+	"github.com/openshift/oc/pkg/cli/deployer/strategy/recreate"
+	"github.com/openshift/oc/pkg/cli/deployer/strategy/rolling"
+	cmdversion "github.com/openshift/oc/pkg/cli/version"
```
Comment (soltysh) [inline staging/src/github.com/openshift/oc/pkg/cli/deployer/deployer.go]: `oc/pkg/version`

Comment (mfojtik) [reply staging/src/github.com/openshift/oc/pkg/cli/deployer/deployer.go]: there are two! :-)

File: pkg/cmd/openshift/openshift.go [hunk 2]
```diff
@@ -15,10 +15,11 @@ import (
 	kcmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
 	ktemplates "k8s.io/kubernetes/pkg/kubectl/util/templates"
 
+	cmdversion "github.com/openshift/oc/pkg/cli/version"
+	osversion "github.com/openshift/oc/pkg/version"
```
Comment (soltysh) [inline pkg/cmd/openshift/openshift.go]: That looks weird, one should be `openshift/origin/pkg/version` and other should be `openshift/oc/pkg/version`.

Comment (mfojtik) [reply pkg/cmd/openshift/openshift.go]: clarification:

- `pkg/version` is where we set the internal vars during build time to commit/version/tag/etc..
- `pkg/cli/version` is the actual `oc version` command :-)

**Pull request merged**